### DISCUSSION TITLE
Media Control Card: Always show all controls

### DISCUSSION THREAD

**Discussion by zsarnett** [home-assistant/frontend#7341] [Dashboard]
Always show the next, pause and prev buttons on the Media Control Card.

Currently, we are hiding certain buttons and features based on the width of the card. We should try and show the controls of the card as long as possible.

Options:

- Just show the controls. Let the image overlap the words

- At a certain width, remove the image and just show the background color.

- Show the controls up until a certain width, then we go back to showing the information of what's playing and the image. More info required to control.

Current Implementation:
![image](https://user-images.githubusercontent.com/18730868/95997847-d797ce00-0df9-11eb-8420-ae85a07d5172.png)

Proposed: (Quick and dirty. Final result would have more style)
![image](https://user-images.githubusercontent.com/18730868/95998731-c8655000-0dfa-11eb-83a6-601c6beb96a7.png)

**Comment by zsarnett** [comment]
Related: #5288 
Related: #5279

**Comment by matthiasdebaat** [comment]
I like your proposal! let's do that
